components/automate-ui/src/app/ngrx.effects.ts

```diff
@@ -42,6 +42,7 @@ import { RuleEffects } from './entities/rules/rule.effects';
 import { RunlistEffects } from './entities/runlists/runlists.effects';
 import { ServerEffects } from './entities/servers/server.effects';
 import { ServiceGroupsEffects } from './entities/service-groups/service-groups.effects';
+import { UserPreferencesEffects } from './services/user-preferences/user-preferences.effects';
```

Comment:
imports must be align in alphabetic order

Reply:
done